### ISSUE TITLE
Show time in properties

### ISSUE THREAD

**Issue by tsknightstorm** [niklaswall/node-red-contrib-light-scheduler#67]
Because time is set often wrong in RPi's it would be very helpful if you can see the time you are based on in the properties panel next to the time table. Then you would recognise errors faster.

**Comment by niklaswall** [comment]
You are right, this is a problem both for Raspberry Pi's but also when running node-red in a docker container (but then mainly timezone issues).

I will investigate what can be done!
